apps/api/lib/api/web/controllers/user_controller.ex

```diff
@@ -46,6 +46,12 @@ defmodule API.Web.UserController do
     end
   end
 
+  def unauthenticated(conn, _params) do
```

Comment:
I was thinking about that but was torn because it felt like adding a new "controller" module just for that was a bit much. The current setup will still apply it to all controllers, but we can split it out if we want, we'll just have to make a new controller